### ISSUE TITLE
Can you please detail how to compile it under Linux? 

### ISSUE THREAD

**Issue by xcom169** [Kromster80/kam_remake#453]
Can you please detail how to compile it under Linux?  Is it in playable state? 
The original Data disc is needed?

**Comment by Kromster80** [comment]
Original Data disc is certainly needed.
We don't have an instruction for Linux though:
 - main development goes in https://github.com/reyandme/kam_remake and Lazarus/FPC support has been deprioritized there AFAIK
 - Wine worked quite for, so we had no reason for native Linus build

**Comment by ghost** [comment]
in my case I need a 32 bit binary so I can emulate it with box86. Ptitseb is developing an arm linux port.. not ready yet... I know that they were some linux binaries somewhere.. could anybody on the team supply them?

**Comment by LauraRozier** [comment]
Not really, there was an attempt to make it work on Linux by compiling from Lazarus but this resulted in MP not working due to how MP checks for client equality (Matching checksums etc). Meaning, no working bins for Linux. an ARM port would therefore also fail, again due to differing checksums.
Until a better client verification method exists you'll sadly have to box the game to be able to play it properly

**Comment by ghost** [comment]
But if single player works... It's a good to go for me.. please, provide that ! If you don't have it.. I am ok, I will wait for the arm port.. or the original game (than I believe it has Linux binaries)

**Comment by TehPlayer14** [comment]
You can compile it someone already did in KaM Remake discord

**Comment by ghost** [comment]
At least I've got the classic version running on my pi through emulation https://youtu.be/fp_qklj0tPk . I hope than ptitseb could port this one soon.

**Comment by TehPlayer14** [comment]
@Askmewho 
Porting Remake wouldn't be easy at all due to PI having an ARM processor and way higher requirements especially AI calculations and pathfinding

**Comment by ghost** [comment]
We will see... Haah

**Comment by xcom169** [comment]
It' easy to run the game .exe inside Wine, but is it possible to load the Game data files without Windows registry?
